### PR TITLE
button animation

### PR DESCRIPTION
**Pull request by AndrewSB**
This depends on #148. The first two commits in here are from the changes there

### CODE REVIEW

**Comment by Kaspik** [comment]
Please rebase.

**Comment by AndrewSB** [comment]
rebased

File: Example/Example/CommonTableViewController.swift [hunk 1]
```diff
@@ -165,7 +165,7 @@ extension CommonTableViewController: InputBarAccessoryViewDelegate {
         tableView.contentInset.bottom = size.height + 300 // keyboard size estimate
     }
     
-    func inputBar(_ inputBar: InputBarAccessoryView, textViewTextDidChangeTo text: String) {
+    @objc func inputBar(_ inputBar: InputBarAccessoryView, textViewTextDidChangeTo text: String) {
```
Comment (Kaspik) [inline Example/Example/CommonTableViewController.swift:168]: Just curious why this? :)

**Review by Kaspik — COMMENTED** [review]
Good to open and merge? One comment :)

File: Example/Example/CommonTableViewController.swift (hunk 1 above)
Comment (AndrewSB) [reply Example/Example/CommonTableViewController.swift]: hmm. good question. I actually don't remember. I think it was an artifact of copying it from my app-code. iirc my class wasn't an NSObject subclass, so this was required.

just removed this and pushed 😄

Comment (AndrewSB) [reply Example/Example/CommonTableViewController.swift]: actually I take it back - looks like it is required. debugging now

Comment (AndrewSB) [reply Example/Example/CommonTableViewController.swift]: Screen Shot 2021-03-05 at 4 42 43 PM

looks like `ButtonAnimationExample.swift` needs it to be marked @objc

Comment (AndrewSB) [reply Example/Example/CommonTableViewController.swift:168]: reverted back to include `@objc`

**Review by Kaspik — APPROVED** [review]

**Pull request merged**